emsdk.py

```diff
@@ -1481,7 +1482,7 @@ def generate_dot_emscripten(active_tools):
 ''' % temp_dir
 
   if embedded:
-    cfg = cfg.replace(emscripten_config_directory, "' + emsdk_path + '")
+    cfg = cfg.replace("'" + emscripten_config_directory, "emsdk_path + '")
```

Comment:
is this intentional? if so, what's going on here?

Reply:
Sorry this is general cleanup on the config.

It changes pattern to:

```
CACHE = emsdk_path + '/.emscripten_cache'
NODE_JS = emsdk_path + '/node/12.9.1_64bit/bin/node'
LLVM_ROOT = emsdk_path + '/fastcomp/fastcomp/bin'
BINARYEN_ROOT = emsdk_path + '/fastcomp'
EMSCRIPTEN_ROOT = emsdk_path + '/fastcomp/emscripten'
EMSCRIPTEN_NATIVE_OPTIMIZER = emsdk_path + '/fastcomp/bin/optimizer'
```

From the more redundant form:

```
CACHE = '' + emsdk_path + '/.emscripten_cache'
NODE_JS = '' + emsdk_path + '/node/12.9.1_64bit/bin/node'
LLVM_ROOT = '' + emsdk_path + '/fastcomp/fastcomp/bin'
BINARYEN_ROOT = '' + emsdk_path + '/fastcomp'
EMSCRIPTEN_ROOT = '' + emsdk_path + '/fastcomp/emscripten'
EMSCRIPTEN_NATIVE_OPTIMIZER = '' + emsdk_path + '/fastcomp/bin/optimizer'
```